config/config.go

```diff
@@ -0,0 +1,29 @@
+/*
+Package config contains edge node manager settings
+*/
+package config
+
+/*
+ProxyPort defines the proxy visor port
+*/
+const ProxyPort string = "3000"
```

Comment:
Why is everything in `config` consts? Surely if it's configuration it can be changed rather than hardcoded like this? What if somebody wants to change a setting, does that require a recompile?